templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -33,7 +33,36 @@
     "Note: LoRA weights will also be stored in path <path>\n",
     "````\n",
     "\n",
-    "This is the path where the adapted weights are stored, you can use them fore inference. You can also see the list of your fine-tuned models in the `serving` tab in the Anyscale console."
+    "This is the path where the adapted weights are stored, you can use them for inference. You can also see the list of your fine-tuned models in the `serving` tab in the Anyscale console.\n",
+    "\n",
+    "### Logging\n",
+    "LLMForge supports logging metrics externally with WandB and MLflow.\n",
+    "\n",
+    "#### WandB\n",
+    "To log with WandB we first need to install the `wandb` library\n",
+    "\n",
+    "```bash\n",
+    "pip install wandb\n",
+    "```\n",
+    "\n",
+    "Then we can launch finetuning by specifying `WANDB_API_KEY`, and specifying `wandb` as the `logger` in our config file. We've provided an example at `training_configs/custom/meta-llama/Meta-Llama-3-8B/lora/4xA10-512-wandb.yaml` with the minimum settings needed to get started logging with WandB.\n",
+    "\n",
+    "```bash\n",
+    "WANDB_API_KEY=<PUT_YOUR_WANDB_KEY_HERE> llmforge anyscale finetune training_configs/custom/meta-llama/Meta-Llama-3-8B/lora/4xA10-512-wandb.yaml\n",
+    "```\n",
+    "\n",
+    "#### MLflow\n",
+    "To alternatively log with MLflow instead of WandB, we can first install MLflow:\n",
+    "```bash\n",
+    "pip install mlflow\n",
+    "```\n",
+    "\n",
+    "Then we can set the `MLFLOW_TRACKING_TOKEN` environment variable instead of `WANDB_API_KEY`, and specify `mlflow` as the `logger` in the config. We've provided an example config at `training_configs/custom/meta-llama/Meta-Llama-3-8B/lora/4xA10-512-mlflow.yaml` showing how to additionally configure the `logger` field with a valid `tracking_uri` and `experiment_name`.\n",
+    "\n",
+    "Then we can launch finetuning as above with our modified config:\n",
+    "```bash\n",
+    "MLFLOW_TRACKING_TOKEN=<PUT_YOUR_MLFLOW_KEY_HERE> llmforge anyscale finetune training_configs/custom/meta-llama/Meta-Llama-3-8B/lora/4xA10-512-mlflow.yaml\n",
+    "```"
```

Comment:
I feel like the intro page here shouldn't start having these feature specific sections. The current way we've organized this is to 
- First document the feature in our intro page in the [docs](https://docs.anyscale.com/llms/finetuning/intro/). 
- Add example configs here if needed
- Add a user guide on the docs which will optionally reference configs here. 
- Make sure there's some visibility to this feature in the template (optional)

One example: https://docs.anyscale.com/llms/finetuning/guides/bring_any_hf_model 


Suggestion:
- This PR will add example configs and just mention the user guide for logging (this note is special for logging, given that there's been a breaking change)
- Update the intro page to highlight new mlflow support 
-  In the user guide, let's update it now and reference the example configs as 

 _We've provided an example config for wandb in the [fine-tuning template](url). To try out fine-tuning with logging on wandb, run
`llmforge anyscale finetune <path_to_config>`_ 


Unfortunately these changes will need to wait until patch release to fix the logger bug (wil push to make it happen soon), but we can keep the material ready.

Reply:
ok so here should I just add the WandB logger to the default config (4xA10-512.yaml), and then we can just add a note mentioning MLflow and linking to the docs

Reply:
Hmm actually the default path can include no logger as well, I think that's fine.

Reply:
But we need to add a note on logger for sure, but it's just going to be linking to the user guide on the docs